{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/config.py

```diff
@@ -109,6 +109,7 @@ def _(x):
 SQLALCHEMY_DATABASE_URI = \
     'mysql+pymysql://{{cookiecutter.project_shortname}}:{{cookiecutter.project_shortname}}@localhost/{{cookiecutter.project_shortname}}'
 {%- endif %}
+#If you are using sqlite, keep SQLALCHEMY_DATABASE_URI undefined and the appropriate file will be created and pointed automatically.
```

Comment:
This should only be added if you are using SQLite.

Reply:
Actually is this comment necessary?

Reply:
I considered that it is not obvious which database is being used since the URI is missing from the config in that case.